proxy/src/main/java/com/velocitypowered/proxy/config/VelocityConfiguration.java

```diff
@@ -88,8 +89,9 @@ private VelocityConfiguration(Servers servers, ForcedHosts forcedHosts, Advanced
   private VelocityConfiguration(String bind, String motd, int showMaxPlayers, boolean onlineMode,
       boolean preventClientProxyConnections, boolean announceForge,
       PlayerInfoForwarding playerInfoForwardingMode, byte[] forwardingSecret,
-      boolean onlineModeKickExistingPlayers, PingPassthroughMode pingPassthrough, Servers servers,
-      ForcedHosts forcedHosts, Advanced advanced, Query query, Metrics metrics) {
+      boolean onlineModeKickExistingPlayers, PingPassthroughMode pingPassthrough,
+      boolean disablePlayerIpLogging, Servers servers,ForcedHosts forcedHosts,
```

Comment:
After giving this a second look, I feel like I shouldn't be changing this constructor. Not sure since I am new to this codebase. Feedback on this would be greatly appreciated :)

Reply:
This is fine 👍